src/resources/i18n/translations/en.json

```diff
@@ -15,13 +15,13 @@
   "Account": "Account",
   "Active Downloads": "Active Downloads",
   "Actual amount will be higher due to network fees": "The actual amount also includes network fees",
-  "Add current item next in queue when playing a new item": "Add current item next in queue when playing a new episode",
-  "Add Custom RSS Feed": "Add Custom RSS Feed",
+  "Add current item next in queue when playing a new item": "Jump the queue when clicking to play new items",
```

Comment:
@comradekingu sorry, "Jump the queue" just doesn't make sense to me here. The currently playing item is not in the "queue" which consists of upcoming items to play, it's in the "player".

This feature takes whatever is the currently playing item in the player, and moves it into the queue when you play a different episode.

It's awkward but "Add current item next in queue when playing a new episode" at least technically correct at least.

Reply:
I got to "Postpone playback to start new items right away" before I reverted to the current version.
"AntennaPod" has "start now" and "play later" as actions when there is something playing.
"Play immediately when something else is playing"?